src/main/java/edu/ucsb/cs156/gauchoride/entities/RideRequest.java

```diff
@@ -0,0 +1,31 @@
+package edu.ucsb.cs156.gauchoride.entities;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.AccessLevel;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "riderequests")
+public class RideRequest {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+  private Long userid;
+  private String day;
+  private String fullName;
```

Comment:
So, this is well intentioned, but a big no-no. 

I know that we need full-name, but we actually already have it—we can get it through the userid.   The userid maps us to an object that includes the users fullname.

Storing redundant data is a big no-no in a relational database app.  It's a common source of bugs.    Since the name is already associated with the user, we don't need to (and should not) store it twice.

Any time you need to display it, you just get this information from the user.

We can also use a ManyToOne and OneToMany mapping so that any time the frontend fetches a ride from the backend, all of the user information comes along with it.  I can show you how when we get to the point where you need that (i.e. when you are building things in the frontend that need the name or email of the rider.)

Reply:
Created new database that doesnt store fullName